interfaces/cython/cantera/test/test_convert.py

```diff
@@ -685,15 +676,19 @@ def test_gri30(self):
         self.checkTransport(ctiPhase, yamlPhase, [298, 1001, 2400])
 
     def test_pdep(self):
-        cti2yaml.convert(Path(self.test_data_dir).joinpath('pdep-test.cti'),
-                         Path(self.test_work_dir).joinpath('pdep-test.yaml'))
+        cti2yaml.convert(
+            self.test_data_path / "pdep-test.cti",
+            self.test_work_path / "pdep-test.yaml"
+        )
         ctiPhase, yamlPhase = self.checkConversion('pdep-test')
         self.checkKinetics(ctiPhase, yamlPhase, [300, 1000, 2200],
                            [100, ct.one_atm, 2e5, 2e6, 9.9e6])
 
     def test_ptcombust(self):
-        cti2yaml.convert(Path(self.cantera_data).joinpath('ptcombust.cti'),
-                         Path(self.test_work_dir).joinpath('ptcombust.yaml'))
+        cti2yaml.convert(
+            self.cantera_data_path / "ptcombust.cti",
+            self.test_work_path / "ptcombust.yaml"
```

Comment:
```suggestion
            self.test_work_path / "ptcombust.yaml",
```

Reply:
And likewise elsewhere, since the closing paren is on the next line.

Reply:
Had already added this in a couple of instances, but absolutely!